src/monitord/monitord.c

```diff
@@ -157,6 +156,18 @@ cJSON *getMonitorInternalOptions(void) {
     return root;
 }
 
+cJSON *getMonitorGlobalOptions(void) {
+
+    cJSON *root = cJSON_CreateObject();
+    cJSON *monconf = cJSON_CreateObject();
+
+    cJSON_AddNumberToObject(monconf,"agents_disconnection_time",Config.agents_disconnection_time);
+    cJSON_AddNumberToObject(monconf,"alert_agent_disconnection_time",Config.alert_agent_disconnection_time);
```

Comment:
In this part of the code, considering that the setting will be saved in memory as seconds, we should add to the JSON `Config.alert_agent_disconnection_time/60` (using the name of the setting suggested in the first comment)

Reply:
Now the user can use a char (_s,m,h,d_) to indicate the unit, but it is safer to return the time in seconds.